plugins/com.google.cloud.tools.eclipse.dataflow.core/src/com/google/cloud/tools/eclipse/dataflow/core/project/GcsDataflowProjectClient.java

```diff
@@ -88,13 +89,16 @@ public StagingLocationVerificationResult createStagingLocation(
     }
   }
 
-  private static String toGcsBucketName(String stagingLocation) {
+  public static String toGcsBucketName(String stagingLocation) {
```

Comment:
This would benefit from some simple javadoc.  `stagingLocation` is presumed to be a GCS URL.

Separately, this method doesn't seem to do anything with leading or trailing whitespace?  Since this is dealing with user input, would seem to be a good thing. Right?